jobs/cc_uploader/spec

```diff
@@ -29,6 +29,10 @@ provides:
     - https_port
 
 properties:
+  set_ulimit_fds:
```

Comment:
please namespace this under `capi.cc_uploader.set_ulimit_fds` and move it lower in the properties hash.

Reply:
Done with commit 3bc3b594c92e6f3b054ea16dd1060593ee7020f1.